cmd/krel/cmd/push.go

```diff
@@ -147,5 +153,89 @@ func init() {
 
 func runPushBuild(opts *pushBuildOptions) error {
 	log.Println("unimplemented")
+
+	var latest string
+	releaseKind := opts.releaseKind
+
+	// Check if latest build uses bazel
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Should we let the errors bubble up instead of exiting via `log.Fatal` here (and at the other places)?

Reply:
Do you mean that we should keep executing? I think it depends on the error. This one I think we should exit immediately because we will be unable to reliably push a build if we cannot get the working dir.

Reply:
What I meant is:
- return an error here instead of `log.Fatal(...)`
- this error should bubble up all the way to be either handled or results in an `os.Exit(...)` or such somewhere centrally.

I would not expect a function that returns an error also to shut down the whole process. If nothing else, that is IMHO not really intuitive and ~hard~ stupid to test.